rules/scala/private/import.bzl

```diff
@@ -89,6 +91,6 @@ def create_intellij_info(label, deps, java_info):
         outputs = java_info.outputs,
         transitive_exports = depset(
             [label],
-            transitive = [(dep[_IntellijInfo] if _IntellijInfo in dep else dep[JavaInfo]).transitive_exports for dep in deps],
+            transitive = [dep[_IntellijInfo].transitive_exports for dep in deps if _IntellijInfo in dep],
```

Comment:
Does IntelliJ still work as expected after this change?

Reply:
Intellij works after this commit https://github.com/lucidsoftware/intellij/commit/0c638b13bc1370143492f0fa2d954c34a5729945
Which some branches of our fork have, and others don't. It also appears to only be compatible with Intellij 2020.3 and above

Reply:
I should rephrase: IntelliJ works fine as is with this change and Bazel 4.2.2, but Bazel 5 only works on later versions.